srcpkgs/recoll/template

```diff
@@ -1,18 +1,21 @@
 # Template file for 'recoll'
 pkgname=recoll
-version=1.30.0
+version=1.32.7
 revision=1
 build_style=gnu-configure
 build_helper=qmake
-configure_args="--disable-python-module --disable-python-chm
- --enable-recollq --disable-x11mon"
-hostmakedepends="automake libtool gettext-devel qt5-qmake pkg-config
- qt5-host-tools"
-makedepends="xapian-core-devel zlib-devel libuuid-devel qt5-webkit-devel qt5-devel
- aspell-devel libxslt-devel libxml2-devel"
+configure_args="--enable-recollq --disable-python-chm --disable-x11mon
+ --disable-webkit --without-systemd"
+hostmakedepends="pkg-config gettext-devel qt5-qmake qt5-host-tools
+ which python3-setuptools python3-devel"
+makedepends="libxslt-devel xapian-core-devel zlib-devel qt5-devel"
```

Comment:
does adding python3-devel to makedepends help with the nocross?

Reply:
based on my tests, it breaks cross under arm8+

Reply:
```
./pyrclextract.cpp:19:10: fatal error: Python.h: No such file or directory
   19 | #include <Python.h>
      |          ^~~~~~~~~~
```
is how aarch64 builds fail with `python3-devel` not in hostmake